list/list.go

```diff
@@ -286,6 +291,18 @@ func (m Model) ShowStatusBar() bool {
 	return m.showStatusBar
 }
 
+// SetStatusBarItemName defines a replacement for the items identifier. Defaults
+// to item/items
+func (m *Model) SetStatusBarItemName(singular, plural string) {
+	m.itemNameSingular = singular
+	m.itemNamePlural = plural
+}
+
+// StatusBarItemName returns the custom status bar title
```

Comment:
Can we add a note here about the return values, I want to make sure people know the order the names come back in
```suggestion
// StatusBarItemName returns singular and plural status bar item names
```